app/Resources/views/themes/ezplatform/form/order.html.twig

```diff
@@ -0,0 +1,5 @@
+{{ form_start(orderForm) }}
```

Comment:
Not a big fan of `orderForm`, it makes me think about an e-commerce website. `sortOrderForm` might be better. What do you think?